addons/purchase/models/purchase_order.py

```diff
@@ -152,6 +152,9 @@ def _compute_invoice(self):
     receipt_reminder_email = fields.Boolean('Receipt Reminder Email', compute='_compute_receipt_reminder_email')
     reminder_date_before_receipt = fields.Integer('Days Before Receipt', compute='_compute_receipt_reminder_email')
 
+    amount_to_invoice = fields.Monetary("Amount to be billed")
+    amount_invoiced = fields.Monetary("Amount already billed")
```

Comment:
It is generalized in account.order, here it's just updating the string to say billed instead of invoiced. Would you prefer if the strings would be the same on the fields in purchase and sales?

Reply:
If it's functionally needed, alrigth :) Maybe just put a small comment on the same line telling that on PO/SO, so that it's more explicit for a reader not paying enough attention as myself :see_no_evil: